en/peripherals/frsky_telemetry.md

```diff
@@ -1,27 +1,65 @@
 # FrSky Telemetry
 
-PX4 supports D (old) and S (new) FrSky telemetry. FMUv4 based boards
-(Pixracer) include the required electronics for the interface, older
-boards (Pixhawk) need a UART to S.PORT adapter board.
+FrSky telemetry allows you to access vehicle telemetry and error information on a compatible RC transmitter.
 
+This information includes: flight mode, battery level, RC signal strength, speed, altitude etc. Some transmitters can additionally provide audible and vibration feedback, which is particularly useful for low battery and other failsafe warnings.
 
-## Pixracer
+## Flight Controller Hardware
 
-> **Success** FrSky telemetry auto-starts on Pixracer and
-  detects D or S mode, there is no need to configure anything.
+Pixhawk/PX4 supports D (old) and S (new) FrSky telemetry. FMUv4 based boards (Pixracer) include the required electronics for the interface, older boards (Pixhawk) need a UART to S.PORT adapter board.
 
-For receivers with D-port (D4R-II and similar): Connect the Pixracer
-FrSky TX line to the D4R-II RX line. Also connect GND.
+### Pixracer
 
-For receivers with S-port: Connect the Pixracer FrSky TX and RX lines
-together (solder the wires together) to the X series receiver's S.port
-pin. Also connect GND.
+#### Configuration
 
-## LuaPilot: Telemetry Screen on the Taranis
-If you have a Taranis that can receive the telemetry stream (e.g. a Taranis X9D
-Plus), you can install a script that shows you a nice overview of the received
-telemetry:
-![](../../images/taranis_telemetry.jpg)
+There is no need to configure anything in PX4 itself. Once connected (as discussed below) FrSky telemetry auto-starts on Pixracer and detects D or S mode. 
+
+#### Connecting Receivers with D-port
+
+Connect the Pixracer FrSky TX line to the receiver's RX line. Also connect GND (This applies to D4R-II and similar).

```

Comment:
@bkueng I think the image below is not associated with this text. Ie it looks to me like this is how you connect the transmitter for normal use (ie not FrSky telemetry). Is that correct? 
For FrSky telemetry in this D-port case I THINK this is saying that you connect the port labeled FSout (TX line) to the Receiver RX line -but we have no image of that. Is that right?

Reply:
yes the image is only for the radio.
I don't think that we have an image. This is how you need to connect for D type:
RX - FS out
TX - FS in
GND: not needed, sincee you already have that from the RC/SBUS connector.
The vast majority today uses S.PORT as far as I can tell.